webextensions/api/webRequest.json

```diff
@@ -29,7 +29,7 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/webRequest/BlockingResponse",
             "support": {
               "chrome": {
-                "version_added": true
+                "version_added": "≤61"
```

Comment:
Should this be 58 re https://github.com/mdn/browser-compat-data/pull/324[)](https://github.com/mdn/browser-compat-data/commit/82b72201e181b8aad847f7904f1c08bda567cb64)